### PR TITLE
Add Yaah to clippy toolstain notification list

### PR DESCRIPTION
**Pull request by yaahc**

### CODE REVIEW

**Comment by yaahc** [comment]
based on @Manishearth 's recommendation

**Review by Manishearth — APPROVED** [review]
@bors delegate+

r=manishearth with that fixed

File: src/tools/publish_toolstate.py [hunk 1]
```diff
@@ -58,8 +58,10 @@ def read_current_status(current_commit, path):
     with open(path, 'rU') as f:
         for line in f:
             (commit, status) = line.split('\t', 1)
+
```
Comment (yaahc) [inline src/tools/publish_toolstate.py]: I have no idea how these spaces got in here o.o

Should I remove them?

Comment (Centril) [reply src/tools/publish_toolstate.py]: Seems like a good idea to not make extra changes.

**Comment by mati865** [comment]
cc https://github.com/rust-lang/rust/pull/64114

Maybe there should be separate `notify` array?

**Comment by pietroalbini** [comment]
Tweaked permissions so any member of the clippy team can be assigned to rust-lang/rust issues.

**Comment by pietroalbini** [comment]
Also opened a PR to add a CI check to ensure maintainers are assignable: https://github.com/rust-lang/rust/pull/64119

**Pull request merged**